knn/README.md

```diff
@@ -0,0 +1,10 @@
+# K-nearest neighbors
+
+Steps to build Music Genre Classification using KNN:
+
+1. Define a function to get the distance between feature vectors and find neighbors
+2. Identify the nearest neighbors
+3. Define a function for model evaluation
+4. Extract features from the dataset and dump these features into a binary .dat file “my.dat”
+5. Train and test split on the dataset
+6. Make prediction using KNN and get the accuracy on test data
```

Comment:
please add more detail of how you do that since other could not understand what you have in the notebook